src/main/groovy/com/adarshr/gradle/testlogger/TestLoggerExtension.groovy

```diff
@@ -10,23 +11,30 @@ import static org.gradle.api.logging.configuration.ConsoleOutput.Plain
 class TestLoggerExtension {
 
     ThemeType theme = STANDARD
-
-    /**
-     * @see org.gradle.api.tasks.testing.logging.TestLogging#getShowExceptions()
-     */
     boolean showExceptions = true
-
     long slowThreshold = 1000
+    boolean showSummary = true
+
+    private final ConsoleOutput consoleType
 
     TestLoggerExtension(Project project) {
+        this.consoleType = project.gradle.startParameter.consoleOutput
         this.theme = project.gradle.startParameter.consoleOutput == Plain ? PLAIN : this.theme
     }
 
     void setTheme(String theme) {
+        if (consoleType == Plain) {
```

Comment:
for me this is failing when we have parallel test enabled. I think that in line 22 we can check if maxParallelForks is greater than 1 and use Plain Parallel instead

Reply:
Yeah. I am inclined to agree now. We should make it a bit more generic and always switch to a parallel theme. Let me think about this again. It also echoes [cjstehno's](#88) thoughts.